.github/workflows/ci.yml

```
@@ -429,4 +428,70 @@ jobs:
+                    path: ~/.nuget/packages
+                    key: ${{ runner.os }}-dotnet10-nuget-${{ hashFiles('**/global.json', '**/Directory.Packages.props', '**/Directory.Build.props', '**/*.csproj', '**/packages.lock.json') }}
+                    restore-keys: |
+                        ${{ runner.os }}-dotnet10-nuget-
+
+            -   name: Restore
+                run: dotnet restore braid.slnx
+
+            -   name: Build
+                run: dotnet build braid.slnx --configuration Release --no-restore
+
+            -   name: Synchronize NDepend project
```

YAMLlint (1.37.1)

[error] too many spaces after hyphen

(hyphens)